components/org.wso2.carbon.identity.auth.service/src/main/java/org/wso2/carbon/identity/auth/service/util/AuthConfigurationUtil.java

```diff
@@ -26,6 +28,7 @@
     private List<String> intermediateCertCNList = new ArrayList<>();
     private List<String> exemptedContextList = new ArrayList<>();
     private boolean isIntermediateCertValidationEnabled = false;
+    private SecretResolver secretResolver;
```

Comment:
should this be a class varibale? can't we move that to the method.

Reply:
> should this be a class varibale? can't we move that to the method.

moved inside the class variable